harpy/_misc.py

```
@@ -84,15 +85,34 @@
+                yml.write(files(harpy.reports).joinpath("_quarto.yml").read_text())
+            else:
+                print_error("report configuration missing", f"The required quarto configuration could not be downloaded from the Harpy repository, nor found in the local file [blue bold]_quarto.yml[/blue bold] that comes with a Harpy installation.")
+                print_solution("There may be an issue with your internet connection or Harpy installation, that latter of which would require reinstalling Harpy. Alternatively, there may be in a issue with your conda/mamba environment or configuration.")
+                sys.exit(1)
+    # same for the scss file
+    try:
+        scss = "https://github.com/pdimens/harpy/raw/refs/heads/main/harpy/reports/_harpy.scss"
+        destination = f"{workdir}/report/_harpy.scss"
+        with urllib.request.urlopen(scss) as response, open(destination, 'w') as scss_out:
+            scss_out.write(response.read().decode("utf-8"))
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)